### ISSUE TITLE
Automatically open `<details>` when linked to anchor/ID inside

### ISSUE THREAD

**Issue by janpio** [fastlane/docs#509]
Example (from https://github.com/fastlane/fastlane/pull/11325):
docs.fastlane.tools/actions/deliver/#available-age-rating-groups
But you first have to open the "Reference" `<details>` element:
docs.fastlane.tools/actions/deliver/#reference
(Then click the first link again)

Possible solution:
https://stackoverflow.com/a/37033774/252627

**Comment by KrauseFx** [comment]
I really like the idea 👎

**Comment by revolter** [comment]
That looks like it works only when you link to the `details` element, and not to an element inside it.

**Comment by janpio** [comment]
Yep, what I posted is only a start. But you could probably go from there - didn't have the time to research more. but wanted to write it down anyway.

**Comment by rogerluan** [comment]
There's also this newer solution in javascript (from March 2019): https://stackoverflow.com/questions/55308339/how-to-expand-details-summary-when-opening-page-with-anchor-inside-nested-detai/55377750#55377750
